pisek/jobs/cache.py

```diff
@@ -142,6 +142,9 @@ def _load(self, env: Env) -> None:
                 return self._new_cache_file()
 
             entries = yaml.full_load(f)
+            if entries is None:
+                return
```

Comment:
Shouldn't this be something like `return self._new_cache_file()`?

Reply:
It doesn't really matter